### PR TITLE
Add snap

### PR DESCRIPTION
**Pull request by tetymd**
I made snapcraft.yaml so that we can use docui with snap.
It is set to build snap based on the latest master's commit.

snapcraft.yaml is used to create a snap locally or on snapcraft.io/build.  
For details of snapcraft format please check here.
https://docs.snapcraft.io/the-snapcraft-format/8337

When pushing master, build and snap are released automatically, 
so I recommend linking repositories with https://snapcraft.io/build.

### Note:
docui needs to set the confinement setting to classic in order to access /var/run/docker.sock of the host OS.  
Also, it seems that it is necessary to obtain approval in the forum to publish snap with classic.
https://docs.snapcraft.io/process-for-reviewing-classic-confinement-snaps/1460
https://forum.snapcraft.io/t/classic-confinement-request-for-the-git-cola-snap/7480

If you do not need formal public release immediately, changing the confinement setting to devmode will allow you to publish snap without authorization from the store.   
In that case, docui can not be found using the "snap search" command.

snapcraft.yaml
```yaml
confinement: devmode # change from classic
```
Install docui
```sh
snap install docui --beta --devmode
or 
snap install docui --edge --devmode
```

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: docui
+version-script: |
+  git describe --tag | awk -F'-([1-9]|g)' '{print $1}'
+version: latest
+summary:  docui is TUI Tool for Docker Client.
```
Comment (skanehira) [inline snap/snapcraft.yaml]: `docui is TUI Client for Docker.`

Comment (tetymd) [reply snap/snapcraft.yaml]: Sure.

**Comment by skanehira** [comment]
@tetymd 
Thank you.
Can you summarize the commit as one?
